addons/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/handler/PLCCommonHandler.java

```diff
@@ -50,9 +51,7 @@
 
     @Nullable
```

Comment:
The pattern is: "modifier @Nullable Type varname;"
Can you change that here, please? :)

Reply:
Strange, now eclipse accept it. Never mind, changed.